pages.de/linux/watch.md

```diff
@@ -14,3 +14,7 @@
 - Überwache "node"-Prozesse und aktualisiere alle 3 Sekunden:
 
 `watch -n {{3}} "{{ps aux | grep node}}"`
+
+- Überwache, ob sich der Festplatten-Speicherplatz ändert und höre mit der Überwachung auf, sobald eine Änderung auftritt:
```

Comment:
Thanks for your contribution. Can you add this to the [English page](https://github.com/tldr-pages/tldr/blob/main/pages/linux/watch.md) too?

Reply:
Sure thing :)

Reply:
Merged it, thanks.